tests/test_tracer.py

```diff
@@ -485,6 +485,80 @@ def test_start_child_from_context():
     eq_(child, child._context._current_span)
 
 
+def test_tracer_hook():
```

Comment:
We should probably have some test coverage for passing args/kwargs with `_emit`.l